src/processor/reconnection-graph.service.ts

```diff
@@ -171,36 +171,35 @@ export class ReconnectionGraphService {
   }
 
   async getUserGraphFromProvider(dsnpUserId: MessageSourceId | string, providerId: ProviderId | string): Promise<any> {
-    const headers = {
-      Authorization: 'Bearer <access_token>', // Replace with your actual access token if required
-    };
-    const baseUrl = this.configService.providerBaseUrl(providerId);
+    const providerAPI = this.getProviderAPI(providerId);
 
     const params = {
       pageNumber: 1,
       pageSize: 10, // This likely should be increased for production values
     };
 
-    const providerAPI: AxiosInstance = axios.create({
-      baseURL: baseUrl.toString(),
-      headers,
-    });
-
     const allConnections: ProviderGraph[] = [];
     const keyPairs: GraphKeyPair[] = [];
-    try {
-      let hasNextPage = true;
-      while (hasNextPage) {
-        // eslint-disable-next-line no-await-in-loop
+
+    let hasNextPage = true;
+    let webhookFailures: number = 0;
+    const webhookFailureThreshold: number = this.configService.getWebhookFailureThreshold();
+    const webhookRetryIntervalMilliseconds: number = this.configService.getWebhookRetryIntervalSeconds() * 1000;
+
+    while (hasNextPage) {
+      // eslint-disable-next-line no-await-in-loop
+      try {
         const response = await providerAPI.get(`/api/v1.0.0/connections/${dsnpUserId.toString()}`, { params });
-        if (response.status !== 200) {
-          throw new Error(`Bad status ${response.status} (${response.statusText} from Provider web hook.)`);
-        }
```

Comment:
I had removed this completely since axios will throw an exception on any non 2xx codes. Other codes in the 200 range may slip through the cracks but aren't expected